springerlink.md

```diff
@@ -1410,8 +1377,7 @@ Pages located at order.springer.com rely solely on built-in browser error checki
 <em>Revised Section 508 - does not apply</em>
 </td>
 <td>Partially Supports</td>
-<td>Status messages that do not receive focus are presented in text. Some are marked up with programmatic roles (e.g. `role="alert"` on some error messages), but there is no consistent, single approach to status messages across the multiple designs.   ARIA Live Regions are not generally used.</td>
-
+<td>Status messages that do not receive focus are presented in text. Some are marked up with programmatic roles (e.g. `role="alert"` on some error messages), but there is no consistent, single approach to status messages across the multiple designs. ARIA Live Regions are not generally used.</td>
```

Comment:
Are we using backticks or `code` elements?

Reply:
Oh, a good point. I generally use backticks as they're slightly easier to read in markdown (imo). Will rationalise throughout